pyproject.toml

```diff
@@ -54,3 +54,6 @@ check_untyped_defs = true
 
 [tool.pylint.SIMILARITIES]
 min-similarity-lines=10
+disable = [
+	"W0621",  # Redefining name %r from outer scope (line %s)
```

Comment:
This is rather annoying for conftest.py, when defining pytest fixtures, for example a fixture named `juju` and then using `juju` as a local variable in another fixture (which is common). So I've turned it off.